ilastik/utility/filemanager.py

```diff
@@ -0,0 +1,36 @@
+import os
+import shutil
+import subprocess
+import sys
+from typing import Union
+
+
+def open(path: Union[str, bytes, os.PathLike]):
```

Comment:
Another possibility is to use `QDesktopServices`
I don't know how reliable it in qt
```python
from PyQt5.QtCore import QUrl
from PyQt5.QtGui import QDesktopServices

QDesktopServices.openUrl(QUrl(f"file://{path}", QUrl.TolerantMode))
```

Reply:
That's awesome, thanks!